backend/app/services/llm/jobs.py

```
@@ -226,6 +261,34 @@ def execute_job(
         )
 
         if response:
+            if output_guardrails:
+                output_text = response.response.output.text
+                safe_output = call_guardrails(output_text, output_guardrails, job_id)
+
+                logger.info(
+                    f"[execute_job] Output guardrail validation | success={safe_output['success']}."
+                )
+
+                if safe_output["bypassed"] == True:
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `safe_output["bypassed"]:` for truth checks

Replace with `safe_output["bypassed"]`

(E712)